projects/core/src/auth/user-auth/services/auth-http-header.service.ts

```diff
@@ -158,6 +159,14 @@ export class AuthHttpHeaderService implements OnDestroy {
     return url.includes(this.occEndpoints.getBaseUrl());
   }
 
+  protected isBaseSitesRequest(request: HttpRequest<any>): boolean {
+    return request.url.includes(
+      this.occEndpoints.getRawEndpointValue('baseSites')
+    )
+      ? true
```

Comment:
I think no need `? true : false`, right?

Reply:
correct, it is redundant, i'll remove them.